packages/plugin-eslint/vite.config.integration.ts

```diff
@@ -10,6 +10,7 @@ export default defineConfig({
     cache: {
       dir: '../../node_modules/.vitest',
     },
+    testTimeout: 10_000,
```

Comment:
Do we know which is the slow test specifically? The timeout could be set for an indivual test instead of all tests at once. 10 seconds isn't so bad though, so no biggie.

Reply:
it was just leftover, thanks for pointing it out